scripts/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -0,0 +1,243 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from __future__ import (absolute_import, division, print_function)
+
+from os import path, makedirs
+from ntpath import basename
+
+from mantid.api import AnalysisDataService as Ads
+from mantid.kernel import logger
+from mantid.simpleapi import Load, EnggVanadiumCorrections, EnggCalibrate, DeleteWorkspace, CloneWorkspace, \
+    CreateWorkspace, AppendSpectra
+from mantidqt.plotting.functions import plot
+from Engineering.EnggUtils import write_ENGINX_GSAS_iparam_file
+
+
+class CalibrationModel(object):
+    def __init__(self):
+        self.VANADIUM_INPUT_WORKSPACE_NAME = "engggui_vanadium_ws"
```

Comment:
If these are just contants, as is indicated by the uppercase variable name?, then the general style is to put them as module-level constants with uppercase names and not have them as variables at the object level.